python/lsst/meas/deblender/baseline.py

```diff
@@ -725,7 +725,7 @@ def newDeblend(debPlugins, footprint, mMaskedImage, psfs, psfFwhms,
     if log is None:
         import lsst.log as lsstLog
 
-        component = 'meas_deblender.baseline'
+        component = 'lsst.meas_deblender.baseline'
```

Comment:
Not sure if you want to switch to using double-quotes here (and elsewhere in this PR), for consistency with other strings? Not a major concern of mine however.

Reply:
Actually. It's not the right logger name. It should be `meas.deblender` and not `meas_deblender` so I fixed it to use `__name__` (Which is what it should have been all along.

Reply:
I'd rather not go through and unify strings everywhere on this PR at this time.

Reply:
Ah, a good move switching to `__name__`, thanks!